semantic.d

```diff
@@ -11010,12 +11023,16 @@ mixin template Semantic(T) if(is(T==ReferenceAggregateDecl)){
 			}
 		}
 	Lfresh:
-		if(!set && decl.stc & STCoverride){
+		if(set) goto Lfresh3;
```

Comment:
Good catch. A better fix is the following:
		OverloadSet set=null;
		if(!parent) goto Lfresh;
		mixin(LookupSealedOverloadSetWithRetry!q{auto setnr; parent, asc, decl.name});
		if(setnr[1]){ needRetry=setnr[1]; return indepvoid; }
		set=setnr[0];